analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -175,26 +174,28 @@ class Bundler : PackageManager() {
                 it?.merge(localGemSpec) ?: localGemSpec
             }
 
+            val gemId = Identifier(javaClass.simpleName, "", gemSpec.name, gemSpec.version)
+
+            // When the project being analyzed is a gem and has a Gemfile that contains the line "gemspec", it will show
+            // up in the list of dependencies. In this case the same VcsInfo that is stored in the Project object should
```

Comment:
Actually, I would expect no package being added at all if the package in fact refers to the project (which is already added)...

Reply:
I agree that this is counterintuitive. It is technically correct when the project itself is listed in the Gemfile (which *is* the list of dependencies), though.

Adding this special case requires much fewer changes than avoiding to add a `Package` and `PackageReference` for the project itself, while still including all of its dependencies and avoiding false positives. However if you have a strong preference for that approach, I'll give it a go.